openshift/tests-extension/.openshift-tests-extension/openshift_payload_olmv1.json

```diff
@@ -1,4 +1,34 @@
 [
+  {
+    "name": "[sig-olmv1][OCPFeatureGate:NewOLMOwnSingleNamespace][Skipped:Disconnected] OLMv1 operator installation support for singleNamespace watch mode with quay-operator should install a cluster extension successfully",
```

Comment:
Same name: https://github.com/openshift/origin/blob/7e8f07d709c533890f1c2f29fddea27fe4acffae/test/extended/util/annotate/generated/zz_generated.annotations.go#L1980

Reply:
We added [Serial] the tests should use this annotation  because they must run in Serial
We must TBD if that is change the annotation with the OTE team
I think it might be an issue since change the name